Libs/MRML/Widgets/qMRMLSliceControllerWidget.cxx

```diff
@@ -781,19 +782,59 @@ void qMRMLSliceControllerWidgetPrivate::updateWidgetFromMRMLSliceNode()
   // Update abbreviated slice view name
   this->ViewLabel->setText(this->MRMLSliceNode->GetLayoutLabel());
 
+  vtkNew<vtkStringArray> orientationNames;
+  this->MRMLSliceNode->GetSliceOrientationPresetNames(orientationNames.GetPointer());
+
+  wasBlocked = this->SliceOrientationSelector->blockSignals(true);
+  int count = this->SliceOrientationSelector->count();
+  if(this->SliceOrientationSelector->itemText(count - 1).compare("Reformat") &&
+     strcmp(this->MRMLSliceNode->GetOrientationString(), "Reformat"))
+    {
+    this->SliceOrientationSelector->removeItem(count - 1);
+    count--;
+    }
```

Comment:
the problem is that is you click on reformat, it will fire setOrientation("Reformat")
which will not change the sliceToRAS matrix, and therefore the ctkCombobox selection it will be automatically ressetted to the last orietationpreset